docs/FAQ.md

```diff
@@ -13,7 +13,174 @@ settings. This is true of all configs in Spark. If you changed
 `spark.sql.autoBroadcastJoinThreshold` after running `explain()` on a `DataFrame`, the resulting
 query would not change to reflect that config and still show a `SortMergeJoin` even though the
 new config might have changed to be a `BroadcastHashJoin` instead. When actually running something
-like with `collect`, `show` or `write` a new `DataFrame` is constructed causing spark to replan the
+like with `collect`, `show` or `write` a new `DataFrame` is constructed causing spark to re-plan the
 query. This is why `spark.rapids.sql.enabled` is still respected when running, even if explain
 shows stale results.
 
+### What versions of Apache Spark does the RAPIDS Accelerator for Apache Spark support?
+
+The RAPIDS Accelerator for Apache Spark requires version 3.0.0 of Apache Spark. Because the plugin
+replaces parts of the physical plan that Apache Spark considers to be internal the code for those
+plans can change even between bug fix releases. As a part of our process we try to stay on top of
+these changes and release updates as quickly as possible.
+
+### Which distributions are supported?
+
+The RAPIDS Accelerator for Apache Spark officially supports
+[Apache Spark](get-started/getting-started.md),
+[Databricks Runtime 7.0](get-started/getting-started-with-rapids-accelerator-on-databricks.md)
+and [Google Cloud Dataproc](get-started/getting-started-gcp.md).
+Most distributions based off of Apache Spark 3.0.0 should work, but because the plugin replaces
+parts of the physical plan that Apache Spark considers to be internal the code for those plans
+can change from one distribution to another. We are working with most cloud service providers to
+set up testing and validation on their distributions.
+
+### What is the right hardware setup to run GPU accelerated Spark?
+
+Reference Architectures should be available around Q4 2020.
+
+### What CUDA versions are supported?
+
+CUDA 10.1 and 10.2 are currently supported, but you need to download the cudf jar that corresponds
```

Comment:
Worth putting a link here, either to the cudf repository or the documentation that discusses how to download cudf?